_tools/fpki-stat-report

```diff
@@ -0,0 +1,113 @@
+**_No Issues_** – No technical or policy issues were found in the last thirty days. </br>
+
+## FPKI Affiliate Status Summary
+
+The following table summarizes the Scorecard status of each connected CA to Federal Common Policy or Federal Bridge.
+
+| Federal Agency or Affiliate CA | FPKIMA CA | Status |
+| ------------------------------ | --------- | ------ |
+| CertiPath Bridge SHA-2 | FBCA | No Issues |
+| Department of Defense SHA-2 | FBCA | No Issues |
+| DigiCert NFI | FBCA | No Issues |
+| Entrust Managed Services NFI | FBCA | No Issues |
+| Exostar NFI | FBCA | No Issues |
+| Government Printing Office | FBCA | No Issues |
+| GSA Access Certificate for Electronic Services (ACES) | FBCA | No Issues |
+| IdenTrust NFI | FBCA | No Issues |
+| WidePoint/ORC NFI | FBCA | No Issues |
+| SAFE BioPharma Bridge | FBCA | No Issues |
+| STRAC Bridge | FBCA | No Issues |
+| TSCP Bridge | FBCA | No Issues |
+| US Patent and Trademark Office (PTO) | FBCA | No Issues |
+| Verizon Business NFI | FBCA | No Issues |
+| CertiPath Bridge SHA-1 | SHA1FRCA | No Issues |
+| Department of Defense SHA-1 | SHA1FRCA | No Issues |
+| Department of State | FCPCA | No Issues |
+| DigiCert/Symantec/Verisign SSP | FCPCA | No Issues |
+| Entrust Managed Services SSP | FCPCA | No Issues |
+| WidePoint/ORC SSP | FCPCA | No Issues |
+| US Treasury SSP | FCPCA | No Issues |
+| Verizon Business SSP | FCPCA | No Issues |
+
+## FPKI Trust Infrastructure Details
+
+### FPKIMA Certificate Activity
+
+The following certificates were issued **BY** or **TO** the FPKI Trust Infrastructure in the last 30 days.
+| Affiliate | Subject CA | Issuing CA | Serial | Issued Date |
+| --------- | ---------- | ---------- | ------ | ------ |
+| No Activity |
+
+The following certificates have been removed from the FPKI Trust Infrastructure in the last 30 days.
+| Affiliate | Subject CA | Issuing CA | Serial | Issued Date |
+| --------- | ---------- | ---------- | ------ | ------ |
+| No Activity |
+
+The following certificates are expiring in the next four months and may be re-issued.
+| Affiliate | Subject CA | Issuing CA | Serial | Expiration Date |
+| --------- | ---------- | ---------- | ------ | ---------- |
+| SAFE BioPharma | SAFE Bridge CA 02 | Federal Bridge CA 2016 | 9 C6 EB AD 66 1E 5B 89 92 84 4D C9 58 57 7E 94 4C DC EF 52 | 06/23/2018 |
+| ORC/WidePoint NFI | ORC NFI CA 3 | Federal Bridge CA 2016 | 3E 67 09 5D 45 33 A2 C9 89 FA 67 3F AB 81 7E D5 58 41 74 22 | 07/31/2018 |
+| SAFE BioPharma | Federal Bridge CA 2016 | SAFE Bridge CA 02 | 74 00 00 00 09 B0 8D 11 8B 00 3E 66 74 00 00 00 00 00 09 | 08/23/2018 |
+| GSA ACES | IdenTrust ACES CA 2 | Federal Bridge CA 2016 | 46 68 3C 1B 51 B5 1C B9 1F 73 97 74 35 E4 3C F8 2B E9 CC 74 | 09/01/2018 |
+| IdenTrust NFI | IdenTrust Global Common Root CA 1 | Federal Bridge CA 2016 | 66 FC 51 BF 22 5B D8 C4 E6 11 E1 56 E7 30 A4 83 FF 99 90 C3 | 09/25/2018 |
+
+### Repository Availability Reporting
```

Comment:
Same suggestion as above - this has been confusing since the table doesn't list all the CAs.  It's only the CAs that are directly signed by Federal Bridge CA or Common?

Reply:
Same comment as above. The intent is to alert readers that the certificates in this specific table may be re-issued and to watch out for a new CA cert.